### PR TITLE
feat(atomic-hosted-page): get ready for deployment

### PR DESCRIPTION
**Pull request by ThibodeauJF**
Let me know if I'm forgetting anything 🕵️ 
https://coveord.atlassian.net/browse/KIT-2266

### CODE REVIEW

File: packages/atomic-hosted-page/package.json [hunk 1]
```diff
@@ -1,7 +1,6 @@
 {
   "name": "@coveo/atomic-hosted-page",
-  "private": "true",
-  "description": "TODO: add description",
+  "description": "Web-component used to inject a Coveo Hosted Search Page in the DOM",
```
Comment (ThibodeauJF) [inline packages/atomic-hosted-page/package.json:3]: For doc, this & the readme, eventually the readme will contain our usage page

File: packages/atomic-hosted-page/README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+# Atomic Hosted Page
+
+A web-component used to inject a Coveo Hosted Search Page in the DOM
```
Comment (jpmarceau) [inline packages/atomic-hosted-page/README.md:3]:
```suggestion
A Web Component used to inject a Coveo Hosted Search Page in the DOM.
```

File: packages/atomic-hosted-page/README.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+```bash
+npm run build
+```
+
+## Run Cypress for Atomic components
+
+Ref: https://docs.cypress.io/
+
+- All the tests will need to be under folder cypress\integration
```
Comment (jpmarceau) [inline packages/atomic-hosted-page/README.md:25]:
```suggestion
- All the tests need to be under folder cypress\integration
```

File: packages/atomic-hosted-page/package.json (hunk 1 above)
Comment (jpmarceau) [inline packages/atomic-hosted-page/package.json:3]:
```suggestion
  "description": "Web Component used to inject a Coveo Hosted Search Page in the DOM.",
```

**Review by jpmarceau — APPROVED** [review]

**Pull request merged**